### ISSUE TITLE
Custom services in Docker

### ISSUE THREAD

**Issue by danielshughes** [bastienwirtz/homer#248]
How can i use the cutsom services e.g. Radarr, Sonarr etc. in Docker?

I can't see the src/components/ folder in the mounted www/assets/ folder and when adding type"Sonarr" to my Sonarr entry it makes it disappear

**Comment by fbartels** [comment]
Yes, that file is indeed not present in the compiled version, you can only see it in the source. Maybe your container is too old and does not yet have the Sonarr plugin?

**Comment by crwallace2** [comment]
I updated to the newest docker container today, and you have to add 2 lines to the config. 
```
type: "Radarr"
apikey: "get API Key from Radar general settings"
```

I am running behind a reverse proxy also so I had to add some headers in the nginx proxy conf file as well to deal with CORS issues. 

   ```
    add_header Access-Control-Allow-Origin https://domain.com;
    add_header Access-Control-Allow-Credentials true;
    add_header Access-Control-Allow-Headers x-api-key;

```

**Comment by Toneti10** [comment]
I am experiencing problems when configuring Sonarr. I am using Docker on DSM 7 (Synology) and homer returns the [following message](https://imgur.com/a/hKDpxEM):

```
Connection error to Sonarr API, check url nad apikey in config.yml
```

My config.yml looks like:

```
- name: "sonarr"
logo: "assets/icons/sonarr.png"
url: "http://192.168.XXX.YYY:ZZZZ" # I have hidden the real numbers
target: "_blank"
type: "Sonarr"
apikey: "xxxxxxxx" # The one I found in Sonarr -> Settings -> General -> Security -> API Key
```

Maybe I need to enter a suffix in the URL? Something additional in Sonarr config? My port number is `ZZZZ` and my Sonarr -> Settings -> General -> Security config is:

```
Authentication: None
API Key: xxxxxxxx
Certificate Validation: Disable for Local Addresses # Was enable, changed in order to try to solve the issue
```

I created the homer container with the following command:

```
docker run -d \
 --name=homer \
 --dns 192.168.AAA.BBB \ # Router IP
 -e PUID=1026 \
 -e PGID=100 \
 -p PPPP:8080 \ # PPPP is a number
 -v /volume1/docker/homer:/www/assets \
 -e TZ=Europe/Madrid \
 --restart always \
b4bz/homer:latest
```

Thanks in advance

**Comment by waschinski** [comment]
Please open the developer tools in your browser and check the console/network tab for more details on the problem you are facing.

**Comment by Toneti10** [comment]
> 
> 
> Please open the developer tools in your browser and check the console/network tab for more details on the problem you are facing.

Thanks! I got the same error as #250 so I'm going to follow up on that issue.

**Comment by Zazou49** [comment]
I have the same error, Error in firefox :

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at ‘http://192.168.1.9:7879/api/queue?apikey=39ac419246934b9eaf49106308c40589’. (Reason: Credential is not supported if the CORS header ‘Access-Control-Allow-Origin’ is ‘*’).`

 https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSNotSupportingCredentials

**Comment by bastienwirtz** [comment]
Hi there!

That's most probably a CORS issue. Check out the [troubleshooting](https://github.com/bastienwirtz/homer/blob/main/docs/troubleshooting.md) section and search for "cors" in past issues. 
I'm closing the issue as the CORS issue is common and documented. Feel free to reopen it if needed.
